CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## unreleased
 
-NOTE:
+NOTE: The signature of UnmarshalTrap now returns an additional error: #394
```

Comment:
I'm not sure what signature means in this context. I think you meant method return?

```suggestion
NOTE: The UnmarshalTrap now returns both an SnmpPacket and an error (#394)
```